css-layout-classes/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header></header>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <aside></aside>
+        </div>
+        <div class="column-two-thirds">
+          <article>
```

Comment:
same as below, this appears to be unrelated HTML, make sure to remove any which is not being used.